extensions/redis-client/runtime/src/main/java/io/quarkus/redis/client/RedisHostsProvider.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.redis.client;
+
+import java.net.URI;
+import java.util.Set;
+
+/**
+ * Programmatically provides redis hosts
+ */
+public interface RedisHostsProvider {
+    /**
+     * Returns the hosts for this provider.
+     * <p>
+     * The host provided uses the following schema `redis://[:password@][host][:port][/database]`
```

Comment:
How does this schema relate to
```
redis://[username:password@][host][:port][/database]
         ^^^^^^^^
```
schema listed at https://quarkus.io/guides/redis#quarkus-redis-client_quarkus.redis.hosts?

Is the `username` actually always fixed to `default`?

Reply:
Good catch, the comment needs to be updated.